src/main/java/org/opentripplanner/graph_builder/DataImportIssuesToHTML.java

```diff
@@ -72,7 +71,7 @@ public void buildGraph(
             //Groups issues in multimap according to issue type
             for (DataImportIssue it : issueStore.getIssues()) {
                 //writer.println("<p>" + it.getHTMLMessage() + "</p>");
-                // writer.println("<small>" + it.getClass().getSimpleName()+"</small>");
+                // writer.println("<small>" + it.getTypeName()+"</small>");
```

Comment:
I think we should remove these lines, isn't it?

Reply:
The issue report could benefit from a "little touch of love", but I did did not want to change to mutch - the PR is mostly about improving the NeTEx import.